hack/update-deps.sh

```diff
@@ -22,6 +22,8 @@ source $(dirname "$0")/../vendor/knative.dev/hack/library.sh
 
 CONTOUR_VERSION="v1.14.0" # This is for controlling which version of contour we want to use.
 
+CLUSTER_ROLE_NAME=contour-knative
```

Comment:
Probably `knative-contour` is better as we have another cluster role `knative-`  prefix?

https://github.com/knative-sandbox/net-contour/blob/12c45bb705cb779b8bb6b4ee73faf36d523c537d/config/200-clusterrole.yaml#L6